wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+# osu!stream
+
+![](img/Os-Logo.jpg "Logo von osu!stream")
+
+**osu!stream** ist eine Version von osu!, die von Grund auf neu entwickelt wurde und extra für berührungssensitive Geräte ausgelegt ist. Aktuell ist sie nur für Apples [iOS](http://en.wikipedia.org/wiki/IOS)-Plattform verfügbar. Ein [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29)-Port war geplant und beinahe fertig, wurde aber nicht veröffentlicht, da es Probleme bei der Audiolatenz gab. Dadurch wurde der Spielspaß massiv beeinträchtigt.
```

Comment:
```suggestion
**osu!stream** ist eine Version von osu!, die von Grund auf neu entwickelt wurde und extra für berührungssensitive Geräte ausgelegt ist. Aktuell ist sie nur für Apples [iOS](http://en.wikipedia.org/wiki/IOS)-Plattform verfügbar. Ein [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29)-Port war geplant und beinahe fertig, wurde aber nicht veröffentlicht, da es Probleme bei der Audiolatenz gab, die den Spielspaß massiv beeinträchtigen.
```

Your version sounds to me like the fact that there is no Android port released is inhibiting enjoyment.

Reply:
I agree with you, but I would choose `beeinträchtigten`, because the rest of the sentence is written in the past tense too.

Reply:
Sure.